apps/docs/pages/guides/auth/server-side/email-based-auth-with-pkce-flow-for-ssr.mdx

```diff
@@ -0,0 +1,236 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Email Auth with PKCE flow for SSR',
+  description:
+    'Learn how to configure email authentication in your server-side rendering (SSR) application to work with the PKCE flow.',
+}
+
+### Install Supabase Auth Helpers
+
+The Auth Helpers will assist you in implementing user authentication within your server-side rendering (SSR) framework.
+
+<Tabs scrollable size="small" type="underlined" defaultActiveId="nextjs">
+  <TabPanel id="nextjs" label="NextJS">
+    ```bash npm install @supabase/auth-helpers-nextjs @supabase/supabase-js ```
+  </TabPanel>
+  <TabPanel id="sveltekit" label="SvelteKit">
+    ```bash npm install @supabase/auth-helpers-sveltekit @supabase/supabase-js ```
+  </TabPanel>
+</Tabs>
```

Comment:
just need to double-check that the formatting here is good - I thought the code blocks need to have no indentation (but I could be wrong)

Reply:
When testing this locally it seemed to work with the indentation.

Reply:
Oh wait it looks like prettier botched that code snippet.